packages/angular/projects/ui-angular/src/lib/services/__tests__/authenticator.service.test.ts

```diff
@@ -1,60 +1,18 @@
 import { AuthenticatorService } from '../authenticator.service';
 
-import * as UIModule from '@aws-amplify/ui';
-import * as XState from 'xstate';
-
-// mock state machine service
-// based on https://github.com/statelyai/xstate/blob/main/packages/core/src/interpreter.ts
-class MockAuthService {
-  private listeners: (() => void)[] = [];
-
-  subscribe(callback: () => void): { unsubscribe: () => void } {
-    this.listeners.push(callback);
-    const unsubscribe = jest.fn();
-    return { unsubscribe };
-  }
-
-  start(): this {
-    return this;
-  }
-
-  send(): void {
-    for (const listener of this.listeners) {
-      listener();
-    }
-  }
-}
-
-const mockFacade = {} as unknown as ReturnType<
-  (typeof UIModule)['getServiceFacade']
->;
-
-jest.spyOn(UIModule, 'getServiceFacade').mockReturnValue(mockFacade);
-// mock interpreted authservice
-jest
-  .spyOn(XState, 'interpret')
-  .mockReturnValue(
-    new MockAuthService() as unknown as ReturnType<typeof XState.interpret>
-  );
-
 describe('AuthenticatorService', () => {
-  let authService: AuthenticatorService;
-
-  beforeEach(() => {
-    authService = new AuthenticatorService();
-    jest.clearAllMocks();
-  });
-
-  it('subscribe returns state machine facade', () => {
-    const handler = jest.fn();
-    const subscription = authService.subscribe(handler);
+  const authService: AuthenticatorService = new AuthenticatorService();
 
+  it('callback is called when interpreted authservice changes', () => {
```

Comment:
We no longer return the facade from the subscribe function, instead it just invokes a callback so testing that that gets called.